cfgov/cfgov/settings/base.py

```diff
@@ -275,12 +275,14 @@
 # Used to include directories not traditionally found,
 # app-specific 'static' directories.
 STATICFILES_DIRS = [
-    PROJECT_ROOT.child("static_built"),
-    PROJECT_ROOT.child("templates", "wagtailadmin"),
+    PROJECT_ROOT.joinpath("static_built"),
+    PROJECT_ROOT.joinpath("templates", "wagtailadmin"),
 ]
 
 # Also include any directories under static.in
-STATICFILES_DIRS += REPOSITORY_ROOT.child("static.in").listdir(filter=DIRS)
+STATICFILES_DIRS += [
+    d for d in REPOSITORY_ROOT.joinpath("static.in").iterdir() if d.is_dir()
+]
```

Comment:
This pattern reaches /static.in/fake-fonts/, but not /static.in/fake-fonts/fonts/, where the files live. 
using glob('**') instead of iterdir() will get there.

I wonder if this could be why we see sporadic missing-font errors?

```suggestion
STATICFILES_DIRS += [
    d for d in REPOSITORY_ROOT.joinpath("static.in").glob('**') if d.is_dir()
]
```

Reply:
@higs4281 I believe this is deliberate and consistent with current behavior on the main branch. Font files are always referred to in code using a `fonts/` prefix, for example [here](https://github.com/cfpb/consumerfinance.gov/blob/701354b2b11675cc6c1c1e0616922d1c07d97ae5/cfgov/jinja2/v1/_layouts/base.html#L134) in our base template or [here](https://github.com/cfpb/design-system/blob/cefdfc3a6402a8551d3ed40effe962553f2d7805/packages/cfpb-typography/src/cfpb-typography.less#L18) in the design system. This means that when we invoke `static('fonts/something')`, that gets added on to the `/static.in/fake-fonts/` path defined here, producing the correct filename.

> we see sporadic missing-font errors

I'm not familiar with these. The custom fonts (Avenir Next) work locally for me. Is there a related issue?

Reply:
No issue.